editor/editor_help.cpp

```diff
@@ -2340,7 +2340,11 @@ void EditorHelp::_help_callback(const String &p_topic) {
 
 	if (class_desc->is_ready()) {
 		// call_deferred() is not enough.
+		if (class_desc->is_connected(SceneStringName(draw), callable_mp(class_desc, &RichTextLabel::scroll_to_paragraph))) {
+			class_desc->disconnect(SceneStringName(draw), callable_mp(class_desc, &RichTextLabel::scroll_to_paragraph));
```

Comment:
I wonder if we really need to disconnect and reconnect again? Or maybe we can just do
`if !connected -> connect`

Reply:
If the `line` is not changed during multiple connect, then yes. I will take a look more closely.

Reply:
I missed the argument binding, indeed. Makes sense, then.